src/main/resources/templates/decisionKnowledgePage.vm

```diff
@@ -12,6 +12,12 @@ $webResourceManager.requireResource("de.uhd.ifi.se.decision.management.jira:deci
 		<li class="menu-item">
 			<a href="#tabs-compare">$i18n.getText('condec.label.compare')</a>
 		</li>
+    <li class="menu-item">
+      <a href="#tabs-relationshipGraph">Relationships</a>
```

Comment:
Can we think of a better name than "Relationships"? Maybe "Decision Relationships" or "Decision/Alternative Relationship"?